src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -147,7 +147,9 @@ private async Task<bool> ExecuteAsync(Common.ILogger log)
 
             if (RestoreNoCache)
             {
-                log.LogWarning(Strings.Log_RestoreNoCacheInformation);
+                //Inform users that NoCache option is just for disabling HttpCache and
+                //suggest them to use NoHttpCache instead, which does the same thing.
+                log.LogInformation(Strings.Log_RestoreNoCacheInformation);
```

Comment:
Can we avoid logging the same thing in [`Program`](https://github.com/NuGet/NuGet.Client/pull/5389/files/4354092afb811f229ff31199df13b7eeeb8375c8..4b5b16770883d57563538b31c1da58dc539f5448#diff-b8b8d924bebf157e6d56ed043f3447d6df86eb521cf75770ea4022908d4903deR136) and here in `RestoreTask`?

Reply:
RestoreTask is not called in NuGet.exe